src/_common/__tests__/Clock-test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import expect from 'expect';
+import { shallow,render } from 'enzyme';
+import { FormattedTime } from 'react-intl';
+import Clock from '../Clock';
+
+describe('Clock',()=>{
+	const wrapper = shallow(<Clock time={ new Date()}> </Clock>);
+    const time = new Date(wrapper.node.props.value);
+	it("The day is Today's day ",()=>{
```

Comment:
The `it` keyword actually refer to the object you want to test, so it's suppose to form a sentence like

`it('should show day as Today's day') `